src/lite-yt-embed.js

```diff
@@ -53,13 +183,9 @@ class LiteYTEmbed extends HTMLElement {
         // Once the user clicks, add the real iframe and drop our play button
         // TODO: In the future we could be like amp-youtube and silently swap in the iframe during idle time
         //   We'd want to only do this for in-viewport or near-viewport ones: https://github.com/ampproject/amphtml/pull/5003
-        this.addEventListener('click', e => this.addIframe());
```

Comment:
kinda surprised this is fine.  i was expecting to lose `this` when the handler was called.

Reply:
How meta is this going to get...

In this case, `this` is the reference to the component. When an event fires, it's against the element as the `this` object. Therefore, `this` is still `this` as expected.

I did test a page with multiple embeds to verify, binding did not seem to have any change in behavior. They all worked as expected, no odd interactions.